data/sql/updates/pending_db_world/rev_1660252072288200163.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` IN (15651, 15652);
+DELETE FROM `smart_scripts` WHERE (`entryorguid` IN (15651, 15652)) AND (`action_param1` = 22863);
```

Comment:
Please add source_type to the condition or you could delete areatriggers or gob SAIs.